packages/pigeon/lib/dart_generator.dart

```diff
@@ -92,6 +92,8 @@ void generateDart(Root root, StringSink sink) {
   final Indent indent = Indent(sink);
   indent.writeln('// $generatedCodeWarning');
   indent.writeln('// $seeAlsoWarning');
+  indent.writeln('// ignore_for_file: public_member_api_docs');
```

Comment:
can you describe what the current issue is?

Reply:
I can't check in the migration of video_player because 'flutter analyze' complains about all the codegen not being documented.  So, we are ignoring that lint.

In the future it would be nice to codegen with documentation.

Reply:
Ah ok, understood. Can we add a todo in the generated code to transfer documentation from the idl? (so we don't forget to come back to this and leave a // ignore forever)

Reply:
https://github.com/flutter/flutter/issues/53191